automated/linux/basic-tests/basic-test-0002_repo-script.yaml

```diff
@@ -0,0 +1,16 @@
+metadata:
+    name: Basic test 0002 - Repo with script
+    format: "Lava-Test Test Definition 1.0"
+    description: "This test checks if RevolutionPi-Repository works and run a simple script."
+    maintainer:
+        - r.gsponer@kunbus.com
+    os:
+        - raspian
+    scope:
+        - functional
+    devices:
+        - Core3_Ostfildern_001
```

Comment:
@ferdrich-kunbus I am not quite sure about how the "devices" here works. But just wonder would it be possible/needed/better to not bind the physical devices with test case ?

Reply:
@zhan-kunbus I have registered my device because that is how Linaro does it in their examples. Nevertheless LAVA does not see any metadata.

Reply:
With different yaml files, it makes the testcase be run on different device ?

Reply:
Files Updated...

Reply:
As far as I know the "devices" is informational for the developers and does nothing for the test itself.